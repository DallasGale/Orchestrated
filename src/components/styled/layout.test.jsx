import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import Layout from './layout'

describe('<Layout />', () => {
  test('it renders', () => {
    const Component = render(<Layout />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
