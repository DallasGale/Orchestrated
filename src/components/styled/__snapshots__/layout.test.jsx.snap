// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> it renders 1`] = `
<div
  class="sc-bdVaJa dqmyKm"
/>
`;
